src/tests/JIT/Regression/JitBlue/Runtime_54842/Runtime_54842.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Runtime.CompilerServices;
+
+public class Runtime_54842
+{
+    public static int Main()
+    {
+        try
+        {
+            DoubleCheckedConvert(uint.MaxValue);
+        }
+        catch (OverflowException)
+        {
+            return 100;
+        }
+
+        return 101;
+    }
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static uint DoubleCheckedConvert(ulong a)
+    {
+        var b = (int)checked((uint)a);
+
+        // Make sure the importer spills "b" to a local.
+        Use(b);
+
+        return checked((uint)b);
+    }
+
+    private static void Use(int value) { }
+}
```

Comment:
Looks like this test was missing a license header, mind if I add it as part of #55989?

Reply:
Not at all, that would be great!